py-polars/polars/internals/frame.py

```diff
@@ -3303,7 +3303,7 @@ def median(self) -> "DataFrame":
         """
         return wrap_df(self._df.median())
 
-    def quantile(self, quantile: float) -> "DataFrame":
+    def quantile(self, quantile: float, interpolation: str = "nearest") -> "DataFrame":
```

Comment:
Let's add a parameters sections which explains the possible interpolation strings